screen_brightness_control/__init__.py

```diff
@@ -621,15 +627,15 @@ def filter_monitor_list(to_filter):
                 # no monitor should be filtered out
                 return to_filter
             elif isinstance(display, int):
-                # 'display' variable should be the index of the monitor
-                # return a list with the monitor at the index or an empty list if the index is out of range
-                return to_filter[display:display + 1]
+                # 'display' variable should match the 'index' field of display info dict
+                # return all displays with matching index
+                return [monitor for monitor in to_filter if monitor['index'] == display]
```

Comment:
For top-level functions the `index` is global scoped. The reason is: if a user calls `sbc.set_brightness(display=0)`, they only expect one monitor to change and not one per method.

So `index=0` usually refers to the first display in the list returned by `list_monitors_info` and not all displays with a method-scoped index of 0.

Since `filter_monitors` is used by `set_brightness` and `get_brightness`, this change should be undone